app/src/main/java/org/mozilla/focus/browser/VideoVoiceCommandMediaSession.kt

```diff
@@ -92,6 +94,7 @@ class VideoVoiceCommandMediaSession @UiThread constructor(
     private var webView: IWebView? = null
     private var sessionIsLoadingObserver: SessionIsLoadingObserver? = null
 
+    private var isResumed = false
```

Comment:
Hm, I wonder if `isInterrupted` is clearer?

Reply:
I think this should stay `isResumed`: see my above comment. I could clarify this in the place where it's used (onPause) by giving it another name, e.g.
```kotlin
fun onPause() {
    val isInterruptedByVoiceCommand = !isResumed
    fun getJS(videoId: String) = if (isInterruptedByVoiceCommand) {
    }
}
```

I'll do that.